### PR TITLE
New alphabetical chapter sort

### LINKED ISSUE #9997: Sort chapters alphabetically
### Describe your suggested feature

When you choose a manga to read you should be able to sort the chapters by alphabetically by the title name, 

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Opi-Txm**
This should close #9997. 
I have only added supporting string values for english and japanese in i18n/src/main/res/values.
I attatched images of the changes below for comparison.

This is my first attempt to contribute to an open source project. If you see any mistakes, please let me know. 
### Images
**Before**
Before
**After**
After

### CODE REVIEW

File: domain/src/main/java/tachiyomi/domain/chapter/service/ChapterSort.kt [hunk 1]
```diff
@@ -23,6 +23,10 @@ fun getChapterSort(
             true -> { c1, c2 -> c2.dateUpload.compareTo(c1.dateUpload) }
             false -> { c1, c2 -> c1.dateUpload.compareTo(c2.dateUpload) }
         }
+        Manga.CHAPTER_SORTING_ALPHABET -> when (sortDescending) {
```
Comment (arkon) [inline domain/src/main/java/tachiyomi/domain/chapter/service/ChapterSort.kt:26]: It's a bit more complicated than this to deal with other languages. You can refer to how [the library does it](https://github.com/tachiyomiorg/tachiyomi/blob/master/app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryScreenModel.kt#L240-L246).

We may want to extract that collator usage out so we can reuse it.

File: app/src/main/java/eu/kanade/presentation/manga/ChapterSettingsDialog.kt [hunk 2]
```diff
@@ -136,6 +136,7 @@ private fun ColumnScope.SortPage(
         R.string.sort_by_source to Manga.CHAPTER_SORTING_SOURCE,
         R.string.sort_by_number to Manga.CHAPTER_SORTING_NUMBER,
         R.string.sort_by_upload_date to Manga.CHAPTER_SORTING_UPLOAD_DATE,
+        R.string.sort_by_alphabet to Manga.CHAPTER_SORTING_ALPHABET,
```
Comment (arkon) [inline app/src/main/java/eu/kanade/presentation/manga/ChapterSettingsDialog.kt:139]: Can reuse `R.string.action_sort_alpha` here. I'll probably reword the other ones separately, they don't necessarily need to be worded like "By xyz".

**Review by arkon — CHANGES_REQUESTED** [review]

File: domain/src/main/java/tachiyomi/domain/manga/model/Manga.kt [hunk 3]
```diff
@@ -85,6 +85,7 @@ data class Manga(
         const val CHAPTER_SORTING_SOURCE = 0x00000000L
         const val CHAPTER_SORTING_NUMBER = 0x00000100L
         const val CHAPTER_SORTING_UPLOAD_DATE = 0x00000200L
+        const val CHAPTER_SORTING_ALPHABET = 0x00000300L
         const val CHAPTER_SORTING_MASK = 0x00000300L
```
Comment (AntsyLich) [inline domain/src/main/java/tachiyomi/domain/manga/model/Manga.kt:89]: this needs to be changed i think not sure to what

Comment (scb261) [reply domain/src/main/java/tachiyomi/domain/manga/model/Manga.kt:89]: I think it's fine? 3(10)=11(2) which covers the new sorting option. The next one (if ever) will require 7(10)=111(2) mask though

**Pull request merged**